torchgeo/datasets/eurosat.py

```diff
@@ -318,3 +318,24 @@ class EuroSAT100(EuroSAT):
         'val': '3e3f8b3c344182b8d126c4cc88f3f215',
         'test': 'f908f151b950f270ad18e61153579794',
     }
+
+
+class SpatialSplitEuroSAT(EuroSAT):
+    """Overrides the default EuroSAT dataset splits.
+
+    Splits the data into training, validation, and test sets based on longitude.
+    The splits are distributed as 60%, 20%, and 20% respectively.
+
+    .. versionadded:: 0.6
+    """
+
+    split_urls = {
+        'train': 'https://hf.co/datasets/torchgeo/eurosat/resolve/1c11c73a87b40b0485d103231a97829991b8e22f/eurosat-spatial-train.txt?download=true',  # noqa: E501
```

Comment:
Doesn't this need to be "raw", not "resolve"? Also, can you remove `?download=true`? I'm also not sure if `noqa` is needed now that we're using ruff.

Reply:
Done